pallets/pallet-asset-switch/src/tests/set_switch_pair.rs

```diff
@@ -44,6 +44,7 @@ fn successful() {
 				Box::new(XCM_ASSET_FEE.into()),
 				u64::MAX as u128,
 				1_000,
+				0,
```

Comment:
Is the zero here intendant? I expected to see a one here.

Reply:
Why?

Reply:
Ah the 1 from this [line](https://github.com/KILTprotocol/kilt-node/pull/669/files/36a43c521ec976baffe1f8a5280cf14a7a81e148#diff-dd0d7d477c7a0ce1248f0b2fa1ada994993b2bc2cc6b45b2979876c1b02420cdR37) is for the ED?

Reply:
Yes, we give `1_000` + the local ED to the pool account. While in this line you commented, `0` is the ED at destination.

Reply:
I see